composer.json

```diff
@@ -25,34 +25,33 @@
         }
     ],
     "require": {
-        "php": ">=7.3.0,<8.1.0",
+        "php": ">=7.4.3,<8.1.0",
         "ext-json": "*",
         "daverandom/exceptional-json": "^1.0.4",
-        "guzzlehttp/guzzle": "^6.5.2 || ^7.0",
-        "monolog/monolog": "^1.25.3 || ^2.0.2",
+        "guzzlehttp/guzzle": "^7.4.0",
+        "psr/log": "^1.1.4 || ^2.0.0 || ^3.0.0",
         "psr/simple-cache": "^1.0.1",
-        "roave/doctrine-simplecache": "^2.2.0",
-        "symfony/console": "^3.4.36 || ^4.4.2 || ^5.0.2",
-        "symfony/filesystem": "^3.4.36 || ^4.4.2 || ^5.0.2"
+        "symfony/console": "^5.4.0",
+        "symfony/filesystem": "^5.4.0"
     },
     "require-dev": {
-        "ergebnis/composer-normalize": "^2.2.2",
-        "friendsofphp/php-cs-fixer": "^2.16.1",
-        "mikey179/vfsstream": "^1.6.8",
-        "pepakriz/phpstan-exception-rules": "^0.11.1",
-        "phpstan/phpstan": "^0.12.9",
-        "phpstan/phpstan-beberlei-assert": "^0.12.2",
-        "phpstan/phpstan-deprecation-rules": "^0.12.2",
-        "phpstan/phpstan-phpunit": "^0.12.6",
-        "phpstan/phpstan-strict-rules": "^0.12.2",
-        "phpunit/phpunit": "^9.5.0"
+        "doctrine/coding-standard": "^9.0.0",
+        "mikey179/vfsstream": "^v1.6.10",
+        "monolog/monolog": "^2.3.5",
+        "phpstan/extension-installer": "^1.1.0",
+        "phpstan/phpstan": "^1.2.0",
+        "phpstan/phpstan-beberlei-assert": "^1.0.0",
+        "phpstan/phpstan-deprecation-rules": "^1.0.0",
+        "phpstan/phpstan-phpunit": "^1.0.0",
+        "phpunit/phpunit": "^9.5.10",
+        "roave/doctrine-simplecache": "^2.4.0"
```

Comment:
We should probably depend on something not deprecated any more, I found [PHP-Cache](http://www.php-cache.com/en/latest/) to be the simplest drop-in replacement for `doctrine/cache`